text/0000-guides-restructure.md

```diff
@@ -0,0 +1,397 @@
+
+We choose the middle ground, because it requires the least new writing, and if we chose to move towards one of the other extremes, it does not prevent that choice nor would we throw away work.
+
+It’s also useful to study the learning flow of other front end libraries in order to determine possible alternatives. Let’s look at a few.
+
+### React
+
+React is known for having low learning overhead for someone who is making their first app. With its popularity, we can guess that new users may expect to find similar topics easily accessible in our guides. This list is most useful for considering what should be in our Components section.
+
+1. Hello World
+2. Introducing JSX
+3. Rendering Elements
+4. Components and Props
+5. State and Lifecycle
+6. Handling Events
+7. Conditional Rendering
+8. Lists and Keys
+9. Forms
+    1. Lifting State Up
+    2. Composition vs Inheritance
+    3. Thinking In React
+
+The Overview section of the React Tutorial also helps show what we may be missing:
+
+- What Is React?
+- Inspecting the Starter Code
+- Passing Data Through Props
+- Making an Interactive Component
+- Developer Tools
+
+Nowhere on our current site do we have a highly visible explanation of what Ember is, beyond snippets. In light of this glaring omission, we have added a “What is Ember” section to the Guides Table of Contents above. It is not meant to replace the ongoing “Why Ember” and marketing-focused descriptions that are underway.
+
+### Vue
+
+As a fully-featured framework, Vue is an easier comparison for possible Table of Contents listings. Keep in mind that much of this type of content is present in our CLI docs instead, so this list will look longer than what we are aiming for.
+
+Introduction
+What is Vue.js?
+Getting Started
+Declarative Rendering
+Conditionals and Loops
+Handling User Input
+Composing with Components
+Relation to Custom Elements
+Ready for More?
+The Vue Instance
+Template Syntax
+Computed Properties and Watchers
+Class and Style Bindings
+Conditional Rendering
+List Rendering
+Event Handling
+Form Input Bindings
+Components Basics
+Components In-Depth
+Component Registration
+Props
+Custom Events
+Slots
+Dynamic & Async Components
+Handling Edge Cases
+Transitions & Animation
+Enter/Leave & List Transitions
+State Transitions
+Reusability & Composition
+Mixins
+Custom Directives
+Render Functions & JSX
+Plugins
+Filters
+Tooling
+Single File Components
+Unit Testing
+TypeScript Support
+Production Deployment
+Scaling Up
+Routing
+State Management
+Server-Side Rendering
+Internals
+Reactivity in Depth
+Migrating
+Migration from Vue 1.x
+Migration from Vue Router 0.7.x
+Migration from Vuex 0.6.x to 1.0
+Meta
+Comparison with Other Frameworks
+Join the Vue.js Community!
+Meet the Team
+
+One possible lesson here is that we could split up Components like Vue did with Component Basics and Components In-Depth. Their dedicated section on Computed Properties inspired the inclusion in our new Table of Contents.
```

Comment:
I'm a fan of this, components can do _a lot_, but all that extra stuff would go in "Leveling Up", which I think is good. starting out, people don't need to manage remote data / queries in their components, but that would be a good topic for a standalone guide rather than in the intro guides. :+1: